packages/pg-pool/index.js

```diff
@@ -136,6 +137,7 @@ class Pool extends EventEmitter {
       const idleItem = this._idle.pop()
       clearTimeout(idleItem.timeoutId)
       const client = idleItem.client
+      client.ref()
```

Comment:
```suggestion
      const client = idleItem.client
      if (this.options.allowExitOnIdle) {
        client.ref()
      }
```

don't we want the symmetry here to match line 331? Otherwise the stream on the connection will be refed an additional time and not unreffed?

Reply:
Ref/unref is a flag, not a counter. The stream comes ref'd by default, so ref'ing it more than once is harmless.

Reply:
ohhh well then! I'll get this released here in a few min